source/RevitLookup/Core/Decomposition/DescriptorsMap.cs

```diff
@@ -55,6 +55,8 @@ public static Descriptor FindDescriptor(object? obj, Type? type)
     {
         return obj switch
         {
+            CurveLoop value when type is null || type == typeof(CurveLoop) => new CurveLoopDescriptor(value),
```

Comment:
Do not use rarely used types at the very beginning of DescriptorsMap, as this will reduce performance. The most frequently used types should be at the top, rarely used ones at the bottom. Move down to the other geometry